src/atomate2/cli/dev.py

```
@@ -602,13 +602,13 @@ def save_abinit_maker(maker: Maker) -> None:
     author_mail = None
     if git:
         name = subprocess.run(
-            "git config user.name".split(),
+            ["git", "config", "user.name"],
```

Ruff (0.8.2)

Starting a process with a partial executable path

(S607)